packages/synapse-react-client/src/lib/containers/widgets/facet-nav/FacetNavPanel.tsx

```diff
@@ -293,35 +279,38 @@ export function FacetPlotLegend(props: FacetPlotLegendProps) {
   }
   const numLegendItems = isExpanded
     ? Math.min(labels.length, 9)
-    : Math.min(labels.length, 3)
+    : Math.min(labels.length, 4)
   if (numLegendItems === 0) {
     return <></>
   }
-  const totalCount = labels.reduce(
-    (curValue, curFacet) => curValue + curFacet.count,
-    0,
-  )
   return (
     <div
       className={`FacetNavPanel__body__legend${isExpanded ? '--expanded' : ''}`}
     >
       {labels.slice(0, numLegendItems).map((facetValue, index) => {
-        const percent = formatPercent(facetValue.count / totalCount, 1)
-        const label = `(${percent}) ${facetValue.label}`
-        const labelDisplay = truncate(label, maxLegendLength)
+        // const percent = formatPercent(facetValue.count / totalCount, 1)
```

Comment:
Is this comment still helpful or should it be dropped since `formatPercent` was [removed](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/161/files#r1177168818)?

Reply:
Definitely should be removed, nice catch